frontend/src/bundles/common/components/stepper/stepper.tsx

```diff
@@ -0,0 +1,59 @@
+import { IconName, Size } from '../../icons/icons.js';
+import { Box, Flex, Header, Icon, IconButton } from '../components.js';
+import { Step } from './components/components.js';
+import { getStepStatus } from './helpers/helpers.js';
+
+type Properties = {
+    steps: string[];
+    currentStep: string;
+};
+
+const stepsElements = ({ steps, currentStep }: Properties): JSX.Element[] => {
+    return steps.map((stepName, index) => {
+        return (
+            <Step
+                key={index}
+                stepDescription={stepName}
+                step={getStepStatus({ stepName, index, currentStep, steps })}
+            />
+        );
+    });
+};
+
+const Stepper: React.FC<Properties> = ({ steps, currentStep }) => {
```

Comment:
Why do you have two different components in one file?
Why do you sometimes use `JSX.Element[]` and sometimes `React.FC<Properties>`?

Reply:
please use React.FC<Properties> instead

Reply:
done